app.py

```diff
@@ -458,10 +460,13 @@ def toggle_create_config_file_modal(_, __, ___):
     trigger = ctx.triggered[0]["prop_id"]
     if trigger == ".":
         raise PreventUpdate
-    elif trigger == "create-config-file-btn.n_clicks":
-        return True
-    elif trigger == "create-config-file-link.n_clicks":
+    #This opens the modal if the user clicks the create config btn or link
+    elif trigger in ["create-config-file-btn.n_clicks", 
```

Comment:
Some [pep8](https://peps.python.org/pep-0008/) issues. I should add a linter to the project (maybe create an issue?). Comments should be <=72 chars long. Or you can use a line break. Also avoid trailing whitespace in 464.